site.yml

```diff
@@ -59,11 +59,11 @@
   - { role: ucarp }
   - { role: docker }
   - { role: etcd, run_as: master }
-  #- { role: ceph-mon, mon_group_name: service-master }
-  #- { role: ceph-osd, mon_group_name: service-master, osd_group_name: service-master }
+  - { role: ceph-mon, mon_group_name: service-master }
+  - { role: ceph-osd, mon_group_name: service-master, osd_group_name: service-master, when: host_capability|match('.*storage.*') }
   - { role: scheduler_stack, run_as: master }
-  - { role: contiv_network, run_as: master }
-  #- { role: contiv_storage, run_as: master }
```

Comment:
let's leave `ceph*` and `contiv_storage` roles commented out in these host-groups until #19 is fixed otherwise it breaks the commission/decommission workflows in `cluster/master`.